src/game/option/option_control.c

```diff
@@ -16,23 +17,17 @@
 #define TOP_Y -60
 #define BORDER 4
 #define HEADER_HEIGHT 25
-#define ROW_HEIGHT 18
+#define ROW_HEIGHT 17
 #define BOX_PADDING 10
 
 #define KC_TITLE -1
 #define COL_END -1
-
-typedef struct LAYOUT_NUM_GS_MAP {
-    INPUT_LAYOUT layout_num;
-    GAME_STRING_ID layout_string;
-} LAYOUT_NUM_GS_MAP;
-
-static const LAYOUT_NUM_GS_MAP m_LayoutMap[] = {
-    { INPUT_LAYOUT_DEFAULT, GS_CONTROL_DEFAULT_KEYS },
-    { INPUT_LAYOUT_CUSTOM_1, GS_CONTROL_CUSTOM_1 },
-    { INPUT_LAYOUT_CUSTOM_2, GS_CONTROL_CUSTOM_2 },
-    { INPUT_LAYOUT_CUSTOM_3, GS_CONTROL_CUSTOM_3 },
-};
+#define COLOR_STEPS 5
+#define RESET_ALL_KEY "R"
```

Comment:
Is this ok to use / will it translate to foreign language keyboards if we ever get to supporting them?

Reply:
I wouldn't worry about these since we don't even support Unicode at the moment.